content/zh/docs/setup/production-environment/turnkey/gce.md

```diff
@@ -0,0 +1,398 @@
+1. 确保将 gcloud 设置成使用你想要的谷歌云平台项目。
+   你可以使用 `gcloud config list project` 检查当前项目，
+   并通过 `gcloud config set project <project-id>` 修改它。
+1. 通过运行 `gcloud auth login`，确保你拥有 GCloud 的凭据。
+1. （可选）如果需要调用 GCE 的 API，你也必须运行 `gcloud auth application-default login`。
+1. 确保你能通过命令行启动 GCE 虚拟机。
+   至少确保你可以完成 GCE 快速入门的[创建实例](https://cloud.google.com/compute/docs/instances/#startinstancegcloud)部分。
+1. 确保你在没有交互式提示的情况下 SSH 到虚拟机。
+   查看 GCE 快速入门的[登录实例](https://cloud.google.com/compute/docs/instances/#sshing)部分。
+
+
+<!-- steps -->
+
+<!-- ## Starting a cluster -->
+## 启动集群
+
+<!-- 
+You can install a client and start a cluster with either one of these commands (we list both in case only one is installed on your machine):
+-->
+你可以安装一个客户端，并使用这些命令的其中之一来启动集群（我们列出的两种情况，因为你的机器可能只安装了二者之一）：
+
+```shell
+curl -sS https://get.k8s.io | bash
+```
+
+或
+
+```shell
+wget -q -O - https://get.k8s.io | bash
+```
+
+<!--  
+Once this command completes, you will have a master VM and four worker VMs, running as a Kubernetes cluster.
+-->
+这条命令结完成后，你将会有 1 个主虚拟机和 4 个工作虚拟机，它们一起作为 Kubernetes 集群运行。
+
+<!--  
+By default, some containers will already be running on your cluster. Containers like `fluentd` provide [logging](/docs/concepts/cluster-administration/logging/), while `heapster` provides [monitoring](https://releases.k8s.io/master/cluster/addons/cluster-monitoring/README.md) services.
+-->
+默认情况下，有一些容器已经在你的集群上运行。
+像 `fluentd` 这样的容器提供[日志记录](/zh/docs/concepts/cluster-administration/logging/)，
+而 `heapster` 提供[监控](https://releases.k8s.io/master/cluster/addons/cluster-monitoring/README.md)服务。
+
+<!--  
+The script run by the commands above creates a cluster with the name/prefix "kubernetes". It defines one specific cluster config, so you can't run it more than once.
+-->
+由上述命令运行的脚本创建了一个名称/前缀为“kubernetes”的集群。
+它定义了一个特定的集群配置，所以此脚本只能运行一次。
+
+<!--  
+Alternately, you can download and install the latest Kubernetes release from [this page](https://github.com/kubernetes/kubernetes/releases), then run the `<kubernetes>/cluster/kube-up.sh` script to start the cluster:
+-->
+或者，你可以通过[这个页面](https://github.com/kubernetes/kubernetes/releases)下载和安装最新版本的 Kubernetes，
+然后运行 `<kubernetes>/cluster/kube-up.sh` 脚本启动集群：
+
+```shell
+cd kubernetes
+cluster/kube-up.sh
+```
+
+<!--  
+If you want more than one cluster running in your project, want to use a different name, or want a different number of worker nodes, see the `<kubernetes>/cluster/gce/config-default.sh` file for more fine-grained configuration before you start up your cluster.
+-->
+如果你希望在项目中运行多个集群，希望使用一个不同名称，或者不同数量工作节点的集群，
+请查看 `<kubernetes>/cluster/gce/config-default.sh` 文件，以便在启动集群之前进行更细粒度的配置。
+
+
+<!--  
+If you run into trouble, please see the section on [troubleshooting](/docs/setup/production-environment/turnkey/gce/#troubleshooting), post to the
+[Kubernetes Forum](https://discuss.kubernetes.io), or come ask questions on `#gke` Slack channel.
+-->
+如果你遇到了问题，请参阅[错误排查](#troubleshooting)一节，
```

Comment:
This is the place where you are referencing the section.

Reply:
yes, I see